android/app/src/main/java/app/organicmaps/widget/menu/NavMenu.java

```diff
@@ -213,14 +213,7 @@ private void updateSpeedView(@NonNull RoutingInfo info)
       return;
 
     Pair<String, String> speedAndUnits = StringUtils.nativeFormatSpeedAndUnits(last.getSpeed());
-
-    if (info.speedLimitMps > 0.0)
-    {
-      Pair<String, String> speedLimitAndUnits = StringUtils.nativeFormatSpeedAndUnits(info.speedLimitMps);
-      mSpeedValue.setText(speedAndUnits.first + "\u202F/\u202F" + speedLimitAndUnits.first);
-    }
-    else
```

Comment:
If you want to remove this part, then, please, check also the bottom of this function. There are some more speed limit checks, f.e.:
* `if (info.speedLimitMps > 0.0 && last.getSpeed() > info.speedLimitMps)`
* `mSpeedViewContainer.setActivated(info.isSpeedCamLimitExceeded());`

`mSpeedViewContainer` has some additional resources/configuration that could be removed/simplified (because it changes color though `setActivated` func)

Reply:
I dind't want to remove the lower parts as they are used to indicate that a speed cam is near. This is not implemented in the new version, so I kept it. And changing the color of the current speed also won't hurt, I plan to create an extra widget for the current speed.
I want to implement the "near radar" warning in a proper banner, but this is still far.